front/public/cgi-bin/printenv.sh

```diff
@@ -0,0 +1,9 @@
+404: Script Not Found
+===================
+
+The script you're looking for was not found OR YOU WERE TRYING TO HACK MY SITE.
+
+NICE TRY. I HAVE LOGGED YOUR IP ADDRESS AND YOU WILL BE HACKED!!1
+
+
+Please report vulnerabilities at [the issue tracker](https://github.com/dikaiosune/rust-dashboard/issues).
```

Comment:
Perhaps we could have a file called security.txt (or similar) and have a message that doesn't make (admittedly humorous) threats? Something like "Please report vulnerabilities to adam.n.perry@gmail.com." ? That could be linked in the index.html the same way you had it set up before.

Sorry, I know I'm being a total stick in the mud.